smurfs/src/actions/index.js

```diff
@@ -1,15 +1,74 @@
-/* 
-  Action Types Go Here!
-  Be sure to export each action type so you can pull it into your reducer
-*/
-
-/*
-  For this project you'll need at least 2 action creators for the main portion,
-   and 2 more for the stretch problem.
-   Be sure to include action types for each type of action creator. Also, be sure to mind
-     the "pending" states like, fetching, creating, updating and deleting.
-   C - addSmurf
-   R - getSmurfs
-   U - updateSmurf
-   D - deleteSmurf
-*/
+// --------------------------- DEPENDANCIES -------------------------------------------//
+import axios from 'axios';
+// --------------------------- DEPENDANCIES -------------------------------------------//
+
+// --------------------------- ACTION EXPORTS -------------------------------------------//
+export const FETCH_SMURFS = 'FETCH_SMURFS';
+export const FETCHED_SMURFS = 'FETCHED_SMURFS';
+export const ADDING_SMURF = 'ADDING_SMURF';
+export const ADDED_SMURF = 'ADD_SMURF';
+export const DELETING_USER = 'DELETING_SMURF';
+export const DELETED_USER = 'DELETED_SMURF';
+export const UPDATING_SMURF = 'UPDATING_SMURF';
+export const UPDATED_SMURF = 'UPDATED_SMURF';
+export const ERROR = 'ERROR';
+// --------------------------- ACTION EXPORTS -------------------------------------------//
+
+// --------------------------- DISPATCHES -------------------------------------------//
```

Comment:
The commenting makes everything super readable and very easy to follow throughout the file.